src/docs/product/integrations/msteams/index.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Microsoft Teams
+sidebar_order: 1
+---
+
+Microsoft Teams is a hub for teamwork in Office 365. Keep all your team's chats, meetings, files, and apps together in one place. Get alerts that let you assign, ignore, and resolve issues right in your Teams channels with the Sentry integration for Microsoft Teams. 
+
+This integration needs to set up only once per organization, then it is available for _all_ projects.
+
+## Install
+
+1. Visit <https://teams.microsoft.com/l/app/5adee720-30de-4006-a342-d454317db1d4>
+
+2. Press the "Add to Team" button and pick the Team you want for the integration
+
+   ![Microsfot Teams installation modal](msteams-add-to-team.png)
+   
+3. You should see a welcome message in the "General" within a few seconds and click "Complete Setup
```

Comment:
I think this should read something like:

You should see a welcome message in the "General" **channel**

(do they call them channels? you get what I mean)